pkg/app/piped/notifier/slack.go

```diff
@@ -170,6 +170,16 @@ func (s *slack) buildSlackMessage(event model.NotificationEvent, webURL string)
 		text = md.Summary
 		generateDeploymentEventData(md.Deployment, md.EnvName)
 
+	case model.NotificationEventType_EVENT_DEPLOYMENT_WAIT_APPROVAL:
+		md := event.Metadata.(*model.NotificationEventDeploymentWaitApproval)
+		title = fmt.Sprintf("Deployment for %q is waiting approve", md.Deployment.ApplicationName)
+		approvers := ""
+		for _, v := range md.Approver {
+			approvers += " <@" + v + ">"
+		}
+		text = fmt.Sprintf("Approval is required from %s", approvers)
```

Comment:
```suggestion
		approvers := make([]string, 0, len(md.Approvers)
		for _, a := range md.Approvers {
			approvers = append(approvers, fmt.Sprintf("<@%s>", a))
		}
		text = fmt.Sprintf("Approvable users: %s", strings.Join(approvers, " "))
```

Reply:
@nghialv 

Thanks! Maybe, `if len(md.Approvers)`? The variable `approvers` is not declared yet at 176th line.

Reply:
You are right. It should be that.